shared/ui-components/src/headers/page_header/page_header.tsx

```diff
@@ -1,6 +1,13 @@
 import * as React from "react"
 
-const PageHeader = (props: any) => {
+interface PageHeaderProps {
+  inverse?: boolean
+  subtitle?: string
+  // TODO: should this be locked down to string?
+  children: any
```

Comment:
@jaredcwhite Ben mentioned that you've been working to lock down props and types. Any thoughts on what we should do for this page header component?

Seems like options are:
1. Specify title as a specific prop, lock it down to string
1. Specify the type of children as something other than any (not sure what) and continue passing in children

Reply:
@akegan Yeah, I think we've been using `React.ReactNode` most places rather than `any` in recent work and that seems to work pretty well (supports a string as well as JSX markup). I'd prefer having the title be inside the component tag rather than use a title prop but that's just my take.

Reply:
ok, updated